package.json

```diff
@@ -1,28 +1,32 @@
 {
-  "name": "raha-api",
-  "version": "1.0.0",
+  "name": "@raha/api-server",
+  "version": "0.0.1",
   "description": "Backend services for Raha.",
   "main": "dist/app.js",
-  "author": "Raha Foundation",
+  "files": [
+    "dist/**/*"
+  ],
+  "author": "Raha",
   "license": "Apache-2.0",
-  "private": true,
+  "private": false,
```

Comment:
P1 - double checking this is intentional ... I guess no harm?

Reply:
Yup intentional. Api will be consumable for its type signtures, constants like URIs and stuff

Reply:
Yup intentional. Api will be consumable for its type signtures, constants like URIs and stuff